core/kernel/eval_spec.rb

```diff
@@ -403,6 +400,7 @@ class EvalSpecs
     end
 
     it "ignores the frozen_string_literal magic comment if it appears after a token and warns if $VERBOSE is true" do
+      default_frozen_string_literal = "test".frozen?
       code = <<CODE
 some_token_before_magic_comment = :anything
 # frozen_string_literal: true
```

Comment:
`default_frozen_string_literal = "test".frozen?` makes the test useless in case String literals are frozen by default.

```suggestion
# frozen_string_literal: #{!default_frozen_string_literal}
```

Reply:
Good point, please do this change

Reply:
It looks like @byroot did this change in https://github.com/ruby/ruby/pull/10262/files#diff-28fcdccc28d1d40986232d2bd551568b9e15faa47964856d08bab83639810f57, so no need to do that, it'll get in this repo.